src/workspace-data/Data.test.ts

```diff
@@ -180,7 +180,7 @@ describe('WorkspaceData', () => {
     });
 
     // Assert
-    expect(screen.getByText(/Preparing your data tables/)).toBeVisible();
+    expect(screen.queryByText(/Preparing your data tables/)).toBeNull(); // cWDS usage removed
```

Comment:
These unit tests DO verify that the changes in this PR are working ... but in the long run, these tests will be pointless - like if we had a test for `expect(screen.queryByText(/This text never existed/)).toBeNull();`.

Your call, I won't block the PR on this - perhaps delete most of these checks and leave one or two that seem the most important?